content/learn/book/getting-started/ecs/_index.md

```diff
@@ -40,6 +40,8 @@ Bevy ECS is Bevy's implementation of the ECS pattern. Unlike other Rust ECS impl
     struct Entity(u64);
     ```
 
+**Note**: You may have noticed that in the above examples, and throughout this tutorial, we use `info!()` from Bevy rather than `println!()` to output text to the console. Why? `info!()` when used in a debug scenario reduces blocking behavior and runs faster compared to `println!()`.
```

Comment:
It would be nice to link to `info` here :)

Reply:
All references in that line, or just the first reference of each type?

Reply:
I think just the first is fine.